crates/red_knot_python_semantic/resources/mdtest/imports.md

```diff
@@ -0,0 +1,133 @@
+# Follow imports
+
+## Classes
+
+We can follow import to class:
+
+````markdown
+```py path=a.py
+from b import C as D; E = D
+reveal_type(E) # revealed: Literal[C]
+```
+
+```py path=b.py
+class C: pass
+```
+````
```

Comment:
Oh, the outer markdown fenced code block is just how the mdtest README handles showing an example markdown document within a markdown README, it's not required in the tests and should be removed here and in all other cases:

````suggestion
```py path=a.py
from b import C as D; E = D
reveal_type(E) # revealed: Literal[C]
```

```py path=b.py
class C: pass
```
````

It's funny that it still works either way, because of our naive regex parsing :)

I'll look at updating the README to address this potential confusion.

Reply:
Fixed, my bad. Once again - thanks for clarifying